vue/components/ui/select-platforme/select-platforme.vue

```diff
@@ -0,0 +1,350 @@
+    border: solid 1px $light-grey;
+    color: $grey;
+}
+
+.select-platforme .dropdown-container .dropdown-button.disabled,
+.select-platforme .dropdown-container .dropdown-button.disabled:active {
+    background-color: $grey-5;
+    border: solid 1px $light-white;
+    color: $medium-grey;
+    margin: inherit;
+}
+
+.select-platforme .dropdown-container .dropdown-button:focus {
+    background-color: $white;
+    border: solid 2px $aqcua-blue;
+    margin: -1px 0px -1px 0px;
+    outline: none;
+}
+
+.select-platforme .dropdown-container ::v-deep .dropdown-platforme {
+    background-color: $white;
+    border: solid 1px $light-white;
+    border-radius: 6px;
+    box-shadow: 0 6px 24px 0 rgba(67, 86, 100, 0.15);
+    margin: 8px 0px 0px 0px;
+    position: absolute;
+}
+
+.select-platforme .dropdown-container ::v-deep .dropdown-platforme .dropdown-item {
+    height: 32px;
+    line-height: 32px;
+    padding: 0px 0px 0px 16px;
+}
+
+.select-platforme .dropdown-container ::v-deep .dropdown-platforme > .dropdown-item > * {
+    padding: 0px 0px 0px 0px;
+}
+
+.select-platforme .dropdown-container ::v-deep .dropdown-platforme > .dropdown-item:hover {
+    background-color: $white;
+}
+
+.select-platforme .dropdown-container ::v-deep .dropdown-platforme .dropdown-item.selectHighlight.optionHighlight,
+.select-platforme .dropdown-container ::v-deep .dropdown-platforme .dropdown-item.selectHighlight {
+    background-color: $grey-3;
+}
+
+.select-platforme .dropdown-container ::v-deep .dropdown-platforme .dropdown-item.optionHighlight {
+    background-color: $light-grey;
+}
+</style>
+
+<script>
+export const SelectPlatforme = {
+    name: "select-platforme",
+    props: {
+        id: {
+            type: String,
+            required: true
+        },
+        options: {
+            type: Array,
+            default: () => []
+        },
+        value: {
+            type: String,
+            default: null
+        },
+        dropdownVisible: {
+            type: Boolean,
+            default: false
+        },
+        placeholder: {
+            type: String,
+            required: true
+        },
+        disabled: {
+            type: Boolean,
+            default: false
+        },
+        allowTextSelection: {
+            type: Boolean,
+            default: false
+        },
+        width: {
+            type: Number,
+            default: 300
+        },
+        maxHeight: {
+            type: Number,
+            default: 200
+        }
+    },
+    data: function() {
+        return {
+            highlightedIndex: null
+        };
+    },
+    watch: {
+        disabled() {
+            if (this.disabled) this.closeDropdown();
+        }
+    },
+    methods: {
+        onClick(event) {
+            const dropdownElement = document.getElementById(`dropdown-${this.id}`);
+            const targetElement = event.target;
+
+            if (!dropdownElement.contains(targetElement)) this.closeDropdown();

```

Comment:
no need for the if

Reply:
removing the if will make it so that if the user tries the open the dropdown, it closes immediately.

Reply:
Replicate the behaviour of the modal: https://github.com/ripe-tech/ripe-components-vue/blob/master/vue/components/ui/modal-platforme/modal-platforme.vue#L236

In that case, you have a inner property (data) and a outer property (visible) to control visibility.